plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -97,6 +97,8 @@ class read_only {
       //string                  recent_slots;
       //double                  participation_rate = 0;
       optional<string>        server_version_string;
+      string                  core_symbol;
+      uint64_t                precision;
```

Comment:
For clarity in JSON, this should be core_symbol_precision.

Reply:
Great idea. This has been updated. Thanks